recipe-box-frontend/src/main/frontend/src/components/RecipesList/Categories.css

```diff
@@ -0,0 +1,18 @@
+button,
+input {
+    border: none;
```

Comment:
When the input is focused, there still is an outline, you should get rid of that